assets/RFC003-AlphaBeta-AtomicSwap.puml

```diff
@@ -0,0 +1,126 @@
+' DIAGRAM #########################################
+' RFC003 atomic swap
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Alice
+participant "Alice\nComit-Node" as AC
+participant "Alpha\nLedger" as Alpha
+participant "Beta\nLedger" as Beta
+participant "Bob\nComit-Node" as BC
+actor Bob
+
+
+' #################################################
+' DIAGRAM
+' #################################################
+Alice->AC: send swap request
+AC->AC: generate secret
+AC->AC: hash secret
+AC->BC: swap request 
+note left #white
+	swap request includes 
+	- hash of secret 
+	- expiries
+	- Alice's identities (if relevant)
+end note
+BC->Bob: new swap request
+Bob->BC: accept swap request
+BC->Alpha: monitor funding of HTLC on Alpha
+BC->AC: accept swap request
+note left #white
+	swap accept includes 
+	- expiries
+	- Bob's identities (if relevant)
+end note
+AC->Alpha: monitor funding of HTLC on Alpha
+AC->Alice: fund action
+Alice->AC: fund Alpha
+AC->AC: prepare HTLC for Alpha
```

Comment:
```suggestion
```
I know you already included a note but actually, this happens the first time when Bob's Comit-Node monitor the funding of HTLC

Reply:
I adapted the complete monitoring part.